BraveRewardsUI/Wallet/Notifications/WalletNotificationView.swift

```diff
@@ -4,16 +4,63 @@
 
 import UIKit
 
+struct WalletNotification {
```

Comment:
If you're going to rename the other from `WalletMessageNotification` to `WalletActionNotification`, why not make this name more specific?

Also why do we now have `alerts`, `actions` and whatever this is? (Message?)

Reply:
We have three different kinds of Notifications. A normal message type, another with action and an alert type. Each has its own style and spacing.
I have made WalletNotificationView as the base view by bringing the common elements into it. Basically code inside WalletNotificationView.commonSetup() is not repeated.

Creating another subclass for Message type